External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -16,6 +16,9 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 %                  masks, since it cannot contain a large data range
 %   bGZip          Gzip the result (OPTIONAL, DEFAULT 1)
 %   changeMat      New orientation matrix 4x4 (OPTIONAL, DEFAULT same as previous)
+%   bSaveJson      Also save a JSON sidecar file with the NIfTI (OPTIONAL, DEFAULT = false)
+%   JsonFields     Additional JsonFields to be added to the new JSON sidecar, by default

```

Comment:
you say that default is pathOriginNifti.json, but then you say DEFAULT = empty. 

Actually, this default should go to bSaveJson description.

Reply:
Fixed in a768f82c2f482e9cdba3e7859302cf4b87dc2de3